src/Gui/CommandView.cpp

```diff
@@ -1633,6 +1633,40 @@ bool StdCmdViewFitSelection::isActive()
   return getGuiApplication()->sendHasMsgToActiveView("ViewSelection");
 }
 
+//===========================================================================
+// Std_ViewCommandGroup
+//===========================================================================
+class StdCmdViewGroup: public Gui::GroupCommand
+{
+public:
+    StdCmdViewGroup::StdCmdViewGroup()

```

Comment:
CI fails because of this line. You are inside the command so you do not need to put `StdCmdViewGroup::StdCmdViewGroup() `but `StdCmdViewGroup()`